### PR TITLE
JST-913160 - Jira/Github integration not showing forked repos

### PR DESCRIPTION
**Pull request by krazziekay**
**What's in this PR?**
- Added `fork:true` qualifier to allow searching for forked repos([source](https://docs.github.com/en/search-github/searching-on-github/searching-in-forks))

**Why**
- One of our customers can not find their forked repo([JST-913160](https://getsupport.atlassian.com/browse/JST-913160))

### CODE REVIEW

File: src/routes/github/repository/github-repository-get.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ const getReposBySubscriptions = async (repoName: string, subscriptions: Subscrip
 				createInstallationClient(subscription.gitHubInstallationId, jiraHost, metrics, logger, subscription.gitHubAppId)
 			]);
 
-			const searchQueryInstallationString = `${repoName} org:${orgName} in:full_name`;
+			const searchQueryInstallationString = `${repoName} org:${orgName} in:full_name fork:true`;
```
Comment (bgvozdev) [inline src/routes/github/repository/github-repository-get.ts:65]: Would it still show regular repos?

**Review by bgvozdev — APPROVED** [review]

File: src/routes/github/repository/github-repository-get.ts (hunk 1 above)
Comment (krazziekay) [reply src/routes/github/repository/github-repository-get.ts:65]: Yup, it should. Tested it quickly locally.
As per the [documentation](https://docs.github.com/en/search-github/searching-on-github/searching-in-forks), there are two options for `fork`:
- `fork:true` that matches all repositories containing the org name including forks.
- `fork:only`that matches all fork repositories containing the org name

**Review by joshkay10 — APPROVED** [review]

**Review by joshkay10 — APPROVED** [review]

File: src/routes/github/repository/github-repository-get.test.ts [hunk 2]
```diff
@@ -148,7 +148,7 @@ describe("GitHub Repository Search", () => {
 
 			const owner = { login: "myOrgName" };
 
-			nockSearchRepos200(`${randomString} org:myOrgName in:full_name`, {
+			nockSearchRepos200(`${randomString} org:myOrgName in:full_name fork:true`, {
 				items: [{ owner, full_name: "first", id: 1 }, { owner, full_name: "second", id: 22 }, { owner, full_name: "third", id: 333 }]
 			});
```
Comment (joshkay10) [inline src/routes/github/repository/github-repository-get.test.ts:154]: Me tooooo down here

**Review by joshkay10 — APPROVED** [review]

File: src/routes/github/repository/github-repository-get.test.ts (hunk 2 above)
Comment (krazziekay) [reply src/routes/github/repository/github-repository-get.test.ts:154]: added

**Pull request merged**